packages/application-components/src/components/dialogs/form-dialog/form-dialog.visualspec.js

```diff
@@ -1,5 +1,12 @@
 import { percySnapshot } from '@percy/puppeteer';
 
+const waitForMs = (ms = 1000) =>
```

Comment:
Nit:

```js
const waitForMs = (ms = 1000) =>
  new Promise(resolve => setTimeout(resolve, ms));
```

Reply:
we don't need this because that test doesn't work anyways. Just remove that from the visualroute and the "click" from the test.